.eslintrc.js

```diff
@@ -50,6 +51,7 @@ module.exports = {
       },
     ],
     '@babel/no-unused-expressions': 'off',
+    'import/consistent-type-specifier-style': 'error',
```

Comment:
```suggestion
    'import/consistent-type-specifier-style': ['error', 'prefer-top-level'],
```

Unlike the following rules, [import/consistent-type-specifier-style](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/consistent-type-specifier-style.md#options) defaults to `prefer-inline`